### PR TITLE
more python examples issue #337

### PR DESCRIPTION
**Pull request by RavirajBaraiya**

### CODE REVIEW

File: examples/amazon-google/AmazonGoogle.py [hunk 1]
```diff
@@ -0,0 +1,64 @@
...
+
+#reading dataset into inputPipe and settint it up in 'args'
+#below line should not be required if you are reading from in memory dataset
+#in that case, replace df with input df
+dfA = spark.read.format("csv").schema("id string, title string, description string, manufacturer string, price double ").load("examples/amazon-google/Amazon.csv")
+dfG = spark.read.format("csv").schema("id string, title string, description string, manufacturer string, price double ").load("examples/amazon-google/GoogleProducts.csv")
+
+inputPipe = []
+
+PipeA = CsvPipe("test")
```
Comment (sonalgoyal) [inline examples/amazon-google/AmazonGoogle.py:29]: Rename a and b to Amazon and google. Also all our variables start with small case, and we follow camel casing

File: examples/amazon-google/AmazonGoogle.py [hunk 2]
```diff
@@ -0,0 +1,64 @@
...
+inputPipe.append(PipeG)
+
+
+args.setData(PipeA)
+args.setData(PipeG)
+
+
+#setting outputpipe in 'args'
+outputPipeG = CsvPipe("resultGoogle")
+outputPipeG.setLocation("/tmp")
```
Comment (sonalgoyal) [inline examples/amazon-google/AmazonGoogle.py:50]: Why do we need two outputs? We just want to do a link

File: examples/iTunes-amazon/iTunesAmazon.py [hunk 3]
```diff
@@ -0,0 +1,51 @@
+from zingg import *
+from zingg.pipes import *
+
+#build the arguments for zingg
+args = Arguments()
+#set field definitions
+id = FieldDefinition("id", "string", MatchType.DONT_USE)
+Song_Name = FieldDefinition("Song_Name", "string", MatchType.FUZZY)
```
Comment (sonalgoyal) [inline examples/iTunes-amazon/iTunesAmazon.py:8]: Same comments as above

File: examples/amazon-google/AmazonGoogle.py [hunk 4]
```diff
@@ -0,0 +1,64 @@
...
+outputPipeA = CsvPipe("resultAmazon")
+outputPipeA.setLocation("/tmp")
+
+
+args.setOutput(outputPipeA)
+
+args.setOutput(outputPipeG)
+
+options = ClientOptions()
+options.setPhase("trainMatch")
```
Comment (sonalgoyal) [inline examples/amazon-google/AmazonGoogle.py:60]: We should set link as the phase

**Review by sonalgoyal — CHANGES_REQUESTED** [review]

**Comment by RavirajBaraiya** [comment]
ncVoter example is not tested successfully because of less RAM

**Comment by RavirajBaraiya** [comment]
@sonalgoyal added new commits for issue #337

**Comment by sonalgoyal** [comment]
Sorry I still see the (C) issue in the data - can you please clean up the
pr?

Cheers,
Sonal
https://github.com/zinggAI/zingg

**Comment by RavirajBaraiya** [comment]
@sonalgoyal unchanged googleProduct.csv

**Comment by sonalgoyal** [comment]
i still see some changes. better would be to remove this file from the pr.
Cheers,
Sonal
https://github.com/zinggAI/zingg

**Pull request closed without merging**